js/src/components/contentAnalysis/Results.js

```diff
@@ -69,11 +70,13 @@ Results.propTypes = {
 	language: PropTypes.string,
 	changeLanguageLink: PropTypes.string,
 	showLanguageNotice: PropTypes.bool.isRequired,
+	canChangeLanguage: PropTypes.bool,
 };
 
 Results.defaultProps = {
 	language: "",
 	changeLanguageLink: "#",
+	canChangeLanguage: false,
```

Comment:
question about `changeLanguageLink` not sure `#` is a proper default value now

Reply:
Merging for now. @Xyfi Please address this when you're back.